src/pm-conversations/GroupPmConversationItem.js

```diff
@@ -1,10 +1,13 @@
 /* @flow strict-local */
-import React from 'react';
+import React, { useContext } from 'react';
 import { View } from 'react-native';
+import { useSelector } from 'react-redux';
 
```

Comment:
```suggestion

import { useSelector } from '../react-redux';
```

Our "type wrapper" gives a bit stronger type checking on uses of `useSelector`, so I've made a habit of using it.

`mutedUsers` does end up with the correct type in this case, but that's because the type of `getMutedUsers` is aware of our `GlobalState`. If we tried to do `const foo = useSelector(state => state.foo)` without using the type wrapper, then I think

1. the `.foo` access wouldn't get an error, and
2. `foo` wouldn't be type-checked anywhere it's used.